scripts/superdesk-authoring/views/change-image.html

```diff
@@ -8,58 +8,85 @@ <h3 class="pull-left">{{ activity.label|translate }}</h3>
         <div class="change-image__metadata" ng-if="data.showMetadataEditor">
             <div class="change-image__metadata__header">Metadata</div>
             <div>
-                <input type="text" placeholder="Add a headline *" ng-model="data.metadata.headline" ng-change="data.isDirty = true">
+                <label>{{'Headline'|translate}} *</label>
+                <input type="text" ng-model="data.metadata.headline" ng-change="data.isDirty = true" class="line-input">
             </div>
             <div>
-                <input type="text" placeholder="Add a description *" ng-model="data.metadata.description_text" ng-change="data.isDirty = true">
+                <label>{{'Description'|translate}} *</label>
+                <input type="text" ng-model="data.metadata.description_text" ng-change="data.isDirty = true" class="line-input">
             </div>
             <div>
-                <input type="text" placeholder="Add a slugline *" ng-model="data.metadata.slugline" ng-change="data.isDirty = true">
+                <label>{{'Slugline'|translate}} *</label>
+                <input type="text" ng-model="data.metadata.slugline" ng-change="data.isDirty = true" class="line-input">
             </div>
             <div>
-                <input type="text" placeholder="Add author / photographer" ng-model="data.metadata.byline" ng-change="data.isDirty = true">
+                <label>{{'Author / photographer'|translate}}</label>
+                <input type="text" ng-model="data.metadata.byline" ng-change="data.isDirty = true" class="line-input">
+            </div>
+            <div>
+                <select ng-model="data.metadata.usageterms">
+                    <option translate value="">---Assing Rights---</option> <!-- not selected / blank option -->
```

Comment:
Author here though would be better translated using the translate directive though.